packages/firebase_dynamic_links/android/build.gradle

```diff
@@ -0,0 +1,37 @@
+group 'io.flutter.plugins.firebasedynamiclinks'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        jcenter()
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        minSdkVersion 16
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+    dependencies {
+        api 'com.google.firebase:firebase-invites:15.0.0'
```

Comment:
Please update this dependency to `15.+` so it stays consistent with other firebase plugins.